apps/cowswap-frontend/src/api/gnosisProtocol/api.ts

```diff
@@ -253,3 +253,26 @@ export function getPriceQuality(props: { fast?: boolean; verifyQuote: boolean |
 
   return PriceQuality.OPTIMAL
 }
+
+export type OrderStatus = {
+  type: 'scheduled' | 'open' | 'active' | 'solved' | 'executing' | 'traded' | 'cancelled'
+  value?: {
+    solver: string
+    sellAmount: number
+    buyAmount: number
+  }[]
+}
+
+// v1/status/<orderId>
+export async function getOrderStatus(chainId: ChainId, orderId: string): Promise<OrderStatus | null> {
+  const response = await fetch('http://localhost:8080/api/v1/status/' + orderId, {
```

Comment:
If you are looking for a solution to deploy some API, Tukan left great setup for us.
Example:
https://github.com/cowprotocol/bff/pull/9
https://cow-web-services-git-feat-nfa-cowswap.vercel.app/api/serverless/nfa

Reply:
Oh that's cool!
It won't be needed though as this is a backend api.
Running it locally because the backend doesn't have any test playground like we do.